staff/dario-fernandez/pre-course/calculator-html/index.js

```diff
@@ -0,0 +1,290 @@
+var boton1 = document.getElementById('n1')
+var boton2 = document.getElementById('n2')
+var boton3 = document.getElementById('n3')
+var boton4 = document.getElementById('n4')
+var boton5 = document.getElementById('n5')
+var boton6 = document.getElementById('n6')
+var boton7 = document.getElementById('n7')
+var boton8 = document.getElementById('n8')
+var boton9 = document.getElementById('n9')
+var boton0 = document.getElementById('n0')
+var botonComa = document.getElementById('n,')
+var botonAC = document.getElementById('AC')
+var botonSuma = document.getElementById('suma')
+var botonResta = document.getElementById('resta')
+var botonMult = document.getElementById('mult')
+var botonDiv = document.getElementById('div')
+var botonIgual = document.getElementById('igual')
+var botonBorrar = document.getElementById('flecha')
+var display = document.getElementById('pantalla')
```

Comment:
It might probably be a better practice to use an initializer that runs after `dom` loaded. Say for some reason, your index.js is moved to the top of the html page, these variables will all be undefined or null. But if you use something like 
`document.addEventListener('DOMContentLoaded', function () {
...
}, false);`

You will always be assured that your variables a initialised with the right values. But in general, this is a pretty neat code. Good job :)

Reply:
Done! Thnx